app/controllers/registrations_controller.rb

```diff
@@ -156,7 +155,7 @@ def do_update(require_password = true, confirm = false)
       set_gettext_locale  #Method defined at controllers/application_controller.rb
       set_flash_message :notice, success_message(_('profile'), _('saved'))
       sign_in current_user, bypass: true  # Sign in the user bypassing validation in case his password changed
-      redirect_to edit_user_registration_path(tab: @tab), notice: success_message(_('profile'), _('saved'))
+      redirect_to edit_user_registration_path, notice: success_message(_('profile'), _('saved'))
```

Comment:
Could you add the hash #personal-details here too?. I'm aware that is the default active tab when the view get rendered but in case we change the default active in future, we will not need to look for this line for when changes at personal details are made.